### DISCUSSION TITLE
How to get the default Tailwind  theme ?

### DISCUSSION THREAD

**Discussion by kvadrakot** [primefaces/community#1277] [PrimeVue] [answered]
I need to use the TailWind theme which is the default on the TailWind website with examples.  In earlier versions this was out of the box, now the situation is more complicated. 

```
import 'primevue/resources/themes/tailwind-light/theme.css';

app.use(PrimeVue, {
    unstyled:false,
});
```
If you implement it this way, then the theme is only partially applied to the components and it does not look like a standard Teilwind
Is it possible to use themes like in regular tailwind?

**Comment by tugcekucukoglu** [comment, marked as answer]
Tailwind is a deprecated theme and we recommend to use new [Tailwind presets](https://tailwind.primevue.org/).
